psp/src/rand.rs

```diff
@@ -0,0 +1,70 @@
+use rand_core::RngCore;
+use core::mem;
+use crate::sys::{self, SceKernelUtilsMt19937Context};
+
+pub struct Mt19937(SceKernelUtilsMt19937Context);
+
+impl Mt19937 {
+    pub fn new(seed: u32) -> Result<Mt19937, ()> { 
```

Comment:
I think it'd be better to use a custom error type here. It won't make much difference now, but being able to add a `Debug` implementation down the road will be nice (`.unwrap()` and `.expect()`)